packages/artifacts/src/download/download.node.ts

```diff
@@ -48,6 +48,17 @@ async function maybeDownload(url: string) {
   return outputPath
 }
 
+/**
```

Comment:
I had first removed them but decided to add them back for reference.

@cedoor do you think we should keep/remove them?

If we have tsdoc comments here, is there a way to "merge" or "include" them in the docs built by the zk-kit repo?

Reply:
Ummh I don't think there's an easy way to include this in the zk-kit doc website. But it could still be good to have comments.